pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -129,6 +132,13 @@ public void init(SegmentGeneratorConfig segmentCreationSpec, SegmentIndexCreatio
       _textIndexColumns.add(columnName);
     }
 
+    for (String columnName : config.getFSTIndexCreationColumns()) {
+        Preconditions.checkState(
+                schema.hasColumn(columnName),
+                "Cannot create text index for column: %s because it is not in schema", columnName);
```

Comment:
Please fix the message

Reply:
ah sorry for the messup on text/fst, should be fixed now